### ISSUE TITLE
[Feature]: bash auto-completion scripts for Enarx

### ISSUE THREAD

**Issue by rjzak** [enarx/enarx#2494]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

It would be nice to have auto-completion working for Enarx with bash, other shells.

**Comment by Ankita13-code** [comment]
Hi @rjzak! Is this issue still relevant? If yes, I would like to give it a try for an auto-completion bash script for enarx commands. Could you let me know how should I get started with this one?

**Comment by rjzak** [comment]
Yes, it's still relevant. As for getting started, I'd look at how bash autocomplete works. Someone once said that the `clap` module has a way to generate this, but I don't know if that's correct, so I'd look into that as well.

Once you have it working, the autocomplete file should be correctly bundled in the resulting .deb and .rpm files, so they're installed and usable when the user installs Enarx.
